db/models/flags/flags.py

```diff
@@ -0,0 +1,43 @@
+"""The module containing all code related to the `flags` table
+within the Postgres db.
+"""
+import uuid
+
+from db import db
+from db.models.flags.enums import FlagType
+from db.models.flags.enums import ResolutionType
+from sqlalchemy import ForeignKey
+from sqlalchemy.dialects.postgresql import ENUM
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.sql import func
+
+
+class Flag(db.Model):
+    """Flag: The sqlalchemy-flask model class used to define the
+    `flags` table in the Postgres database."""
+
+    __tablename__ = "flags"
+
+    id = db.Column(
+        "flag_id", UUID(as_uuid=True), default=uuid.uuid4, primary_key=True
+    )
+
+    justification = db.Column("justification", db.Text(), nullable=False)
+
+    section_to_flag = db.Column("section_to_flag", db.Text(), nullable=False)
+
+    flag_type = db.Column("flag_type", ENUM(FlagType), nullable=False)
+
+    resolution_reason = db.Column(
+        "resolution_reason", ENUM(ResolutionType), nullable=True
+    )
+
+    application_id = db.Column(
```

Comment:
just wondering if there is a restraint that there should only ever be one flag per application at a time? Unsure on the business rules around this so more of a Q tbf

Reply:
Hey, good point!  From what I could gather, there _might_ be different types of flagging in the future... so maybe something for the future?